CMakeLists.txt

```diff
@@ -172,18 +172,22 @@ function(RELATIVE_PROTOBUF_GENERATE_CPP SRCS HDRS ROOT_DIR)
              "${OUTPUT_PB_HEADER}"
       COMMAND ${ONNX_PROTOC_EXECUTABLE}
       ARGS ${PROTOC_ARGS}
-      DEPENDS ${GENERATED_PROTO}
+      DEPENDS ${GENERATED_PROTO} ${DEPEND}
       COMMENT "Running C++ protocol buffer compiler on ${GENERATED_PROTO}"
       VERBATIM )
+  add_custom_target(${NAME}
+      DEPENDS ${OUTPUT_PB_SRC} ${OUTPUT_PB_HEADER})
   endforeach()
 
   set_source_files_properties(${${SRCS}} ${${HDRS}} PROPERTIES GENERATED TRUE)
   set(${SRCS} ${${SRCS}} PARENT_SCOPE)
   set(${HDRS} ${${HDRS}} PARENT_SCOPE)
 endfunction()
 
-RELATIVE_PROTOBUF_GENERATE_CPP(PROTO_SRCS PROTO_HDRS ${ONNX_ROOT}
+RELATIVE_PROTOBUF_GENERATE_CPP(gen_onnx_proto PROTO_SRCS PROTO_HDRS ${ONNX_ROOT} ""
     onnx/onnx.in.proto
+)
+RELATIVE_PROTOBUF_GENERATE_CPP(gen_onnx_operators_proto PROTO_SRCS PROTO_HDRS ${ONNX_ROOT} gen_onnx_proto 
```

Comment:
PROTO_SRCS got overridden to only have the onnx_operators.pb.cc now, this is the root cause in the current CI linkage error

Reply:
oh!! my mistake, sorry!
I will fix this!